packages/admin/cms-admin/src/warnings/warningMessages.tsx

```diff
@@ -2,4 +2,10 @@ import { FormattedMessage } from "react-intl";
 
 export const warningMessages = {
     missingHtmlTitle: <FormattedMessage id="comet.warnings.missingHtmlTitle" defaultMessage="Missing HTML Title" />,
+    invalidInternalLinkTarget: (
+        <FormattedMessage id="comet.warnings.internalLinkTargetPageInvalid" defaultMessage="Invalid Target in Page Tree Link" />
+    ),
+    missingInternalLinkTarget: (
+        <FormattedMessage id="comet.warnings.internalLinkTargetPageNotDefined" defaultMessage="Missing Target in Page Tree Link" />
+    ),
```

Comment:
maybe that could be simplified:
```suggestion
    invalidTarget: (
        <FormattedMessage id="comet.warnings.invalidTarget" defaultMessage="Invalid Target" />
    ),
    missingTarget: (
        <FormattedMessage id="comet.warnings.missingTarget" defaultMessage="Missing Target" />
    ),
```

- messages are generic, no need to include internalLink in the key/id
- I would skip "in Page Tree Link" as that doesn't give any additional context